src/App.tsx

```diff
@@ -1,24 +1,230 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { Todo } from './types/Todo';
+import {
+  deleteTodo,
+  getTodos,
+  patchTodo,
+  postTodo,
+} from './api/todos';
+import { TodoList } from './components/TodoList';
+import { TodoFilter } from './components/TodoFilter';
+import { Notification } from './components/Notification';
+import { StatusTodos } from './types/StatusTodo';
 
-const USER_ID = 0;
+const USER_ID = 9946;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [status, setStatus] = useState(StatusTodos.ALL);
+  const [query, setQuery] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+  const [activeTodoId, setActiveTodoId] = useState<number>(0);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const isTodoCompleted = todos.every(todo => todo.completed);
+
+  const filteredTodos = useMemo(() => todos.filter(todo => {
+    switch (status) {
+      case StatusTodos.ACTIVE:
+        return !todo.completed;
+
+      case StatusTodos.COMPLETED:
+        return todo.completed;
+
+      default:
+        return StatusTodos.ALL;
+    }
+  }), [status, todos]);
+
+  const loadTodos = async () => {
+    try {
+      const todosData = await getTodos(USER_ID);
+
+      if ('Error' in todosData) {
+        throw new Error('Error in TodosData');
+      } else {
+        setTodos(todosData);
+      }
+    } catch (error) {
+      setErrorMessage("Can't create a todo");
```

Comment:
Better to change the error message to "Can't load todos" to accurately reflect the failed operation.

You can also create an enum for error messages